public/js/themeMetadata.js

```diff
@@ -10,5 +10,10 @@ const theme = {
     "No2TypeB": {
         "css": "/css/themes/No2TypeB.css",
         "key": "No2TypeB"
-    }
+    },
+    "SpaceGroove": {
+        "css": "/css/themes/SpaceGroove.css",
+        "key": "SpaceGroove",
+        "videoSource": "/static/spacegroove.mp4"
```

Comment:
Shouldn't spacegroove.mp4 be committed via LFS? We've had problems with this repo ballooning from media and other items before... (See za419/Ken-archival-cadence)

Reply:
I thought about LFS but strongly prefer to not make LFS installation a requirement of the project again. I would ideally like to have it served on a third party and considered some options:

- Use Youtube as the video host at the cost of having to use an iframe with a Youtube watermark.
- Use Imgur as the video host as I do with most other Cadence media, at the cost of a maximum 720p resolution.
- Use object storage in Amazon at the risk of egregious egress fees on the order of tens of thousands of dollars in the case someone decides to load up the video a few million times.
- Use object storage in Cloudflare who recently announced R2, which has no egress fees, but have not released it yet.

I would like to merge as it is only one video for now, and make it a pending issue to offload media into Cloudflare R2 once it releases, then clean the repo of large files as I did before.

Reply:
Perhaps for now we could have a separate repo storing large files with a gitignore and a link on the server?

I'm thinking it'd be best to avoid the need to push through a filter to begin with...

Reply:
I am attempting to use a static submodule here: #192 No gitignore is required with the submodule to filter relevant media, as the cadence repository will see it as a single submodule "file" that is independently updated.

This does add a requirement to run `git submodule update --recursive` when new media is added. However, I feel it is acceptable as the submodule can be removed easily in the future like any file, which should keep this repository clean.

Reply:
Well, I _believe_ the size should not balloon. I need to run a test.

Reply:
Ok. It will not balloon the size for new repo cloners: https://github.com/kenellorando/submodule-size-test/blob/master/README.md